addons/ccm-openstack/ccm-openstack.yaml

```diff
@@ -166,40 +145,31 @@ spec:
         "kubeone.k8c.io/cabundle-hash": "{{ .Config.CABundle | sha256sum }}"
         "kubeone.k8c.io/cloudconfig-hash": "{{ .Config.CloudProvider.CloudConfig | sha256sum }}"
       labels:
-        k8s-app: "openstack-cloud-controller-manager"
+        component: controllermanager
+        app: openstack-cloud-controller-manager
+        release: openstack-ccm
+        chart: openstack-cloud-controller-manager-2.27.1
+        heritage: Helm
     spec:
-      priorityClassName: system-cluster-critical
-      nodeSelector:
-        node-role.kubernetes.io/control-plane: ""
       securityContext:
         runAsUser: 1001
-      tolerations:
-        - key: "node.cloudprovider.kubernetes.io/uninitialized"
-          value: "true"
-          effect: "NoSchedule"
-        - key: "node-role.kubernetes.io/master"
-          effect: NoSchedule
-        - key: "node-role.kubernetes.io/control-plane"
-          effect: NoSchedule
-      serviceAccountName: cloud-controller-manager
+      serviceAccountName: openstack-cloud-controller-manager
       containers:
         - name: openstack-cloud-controller-manager
           image: {{ .InternalImages.Get "OpenstackCCM" }}
-          command:
+          args:
             - /bin/openstack-cloud-controller-manager
             - --v=1
-            - --cloud-config=/etc/config/cloud-config
+            - --cloud-config=$(CLOUD_CONFIG)
+            - --cluster-name=$(CLUSTER_NAME)
             - --cloud-provider=openstack
             - --use-service-account-credentials=true
+            - --controllers=cloud-node,cloud-node-lifecycle,route,service
             - --bind-address=127.0.0.1
-            {{ if .CCMClusterName }}
-            - --cluster-name={{ .CCMClusterName }}
-            {{ end }}
-{{ if .Config.CABundle }}
-          env:
-{{ caBundleEnvVar | indent 12 }}
-{{ end }}
           volumeMounts:
```

Comment:
This is missing volumeMounts for custom CA.

Reply:
done